isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -169,6 +169,111 @@ namespace isobus
 		}
 	}
 
+	void VirtualTerminalClient::RegisterVTESCMessageEventCallback(VTESCMessageCallback value)
+	{
+		escMessageCallbacks.push_back(value);
+	}
+
+	void VirtualTerminalClient::RemoveVTESCMessageEventCallback(VTESCMessageCallback value)
+	{
+		auto callbackLocation = find(escMessageCallbacks.begin(), escMessageCallbacks.end(), value);
```

Comment:
Nit: `find` is in `std::` namespace I think, so ideally we should prefix with `std::` in these functions